### DISCUSSION TITLE
求助

### DISCUSSION THREAD

**Discussion by lbnliboning** [lsky-org/lsky-pro#623] [问答]
![15198BDA-1EB7-48F8-A6B9-EE493D15EFDC.png](https://bu.dusays.com/2023/02/26/63fb3e8a2f6a5.![4ACD5A60-564C-4A45-B384-D9BEEF2CC6C2](https://user-images.githubusercontent.com/106464318/221407052-0911eceb-a562-41fc-a7e6-949f0ff23eeb.png)

**Comment by lbnliboning** [comment]
https://bu.dusays.com/2023/02/26/63fb3e8a2f6a5.jpge

**Comment by lbnliboning** [comment]
Warning: require(/www/wwwroot/img.1703.site/public/../vendor/autoload.php): Failed to open stream: No such file or directory in /www/wwwroot/img.1703.site/public/index.php on line 34

Fatal error: Uncaught Error: Failed opening required '/www/wwwroot/img.1703.site/public/../vendor/autoload.php' (include_path='.:') in /www/wwwroot/img.1703.site/public/index.php:34 Stack trace: #0 {main} thrown in /www/wwwroot/img.1703.site/public/index.php on line 34

**Comment by WuMe-sicx** [comment]
请问？是什么问题呢
